ci/travis/36-pd023.yaml

```diff
@@ -1,6 +1,6 @@
 name: test
 channels:
-  - conda-forge
+  - defaults
```

Comment:
It should work. `defaults` uses only `gdal 2.3` at the moment and they did not update the dependencies yet.